android/src/ti/iap/TitaniumInAppPurchaseModule.kt

```diff
@@ -10,15 +10,18 @@
 
 package ti.iap
 
-import android.util.Log
 import com.android.billingclient.api.*
 import com.android.billingclient.api.BillingClient.BillingResponseCode.*
 import com.android.billingclient.api.BillingClient.FeatureType.*
 import com.android.billingclient.api.BillingClient.SkuType.INAPP
 import com.android.billingclient.api.BillingClient.SkuType.SUBS
+import com.android.billingclient.api.BillingFlowParams.ProrationMode
```

Comment:
If you remove that (deprecated and removed in 7) you can also upgrade to `inapp_version = "7.0.0"`
https://developer.android.com/google/play/billing/release-notes#google_play_billing_library_700_release_2024-05-14
builds fine but I didn't do any tests with the module

Reply:
It was actually replaced, so I'd look into checking the new API